### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by codeofcoc** [iTODDLERS-BTFO/iToddlers-BTFO#1]
I just couldn't help but notice there is no Code of Conduct (CoC) for this amazing project. I am sure this was just an oversight and not intentional. Personally I recommend the Rust programing language CoC. You can read it here: https://prev.rust-lang.org/en-US/conduct.html

Thanks!
 codeofcoc

**Comment by erikKeresztes** [comment]
i agree, we must fight transphobia, homophobia, pocphobia, and jewphobia

**Comment by HawaiinPizza** [comment]
yo what about brownskinphobia? We Mexicans, Arabs and southeast Asians are discriminated against too

**Comment by awsms** [comment]
based and code of conductpilled

**Comment by Benjababe** [comment]
We must not let the racists, transphobes, bigots and antisemites larp in the cause of btfo-ing itoddlers!
